recipe/meta.yaml

```diff
@@ -80,125 +80,9 @@ outputs:
     test:
       commands:
         - duckdb -version
-  - name: duckdb-extension-json
```

Comment:
In your PR description you mention dropping the fts and https extensions.

> This PR drops the extensions fts and https conda-forge packages. The automatic update failed because these extensions moved out-of-tree.

But the diff suggests that you drop a lot more extensions than that. What is your plan with these extensions?

Reply:
JSON is now a built-in extension and will no longer be separated installed.